plat/common/arm/lcpu.c

```diff
@@ -58,7 +58,7 @@
  *                     SP_EL0
  */
 static __align(UKARCH_SP_ALIGN)
-UKPLAT_PER_LCPU_ARRAY_DEFINE(__u8, lcpu_irqntrap_sp, STACK_SIZE * 2);
+UKPLAT_PER_LCPU_ARRAY_DEFINE(__u8, lcpu_irqntrap_sp, CPU_EXCEPT_STACK_SIZE * 2);
```

Comment:
Must also update `exceptions.S` for IRQ entries to add `CPU_EXCEPT_STACK_SIZE` instead of `STACK_SIZE`